src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -184,6 +193,28 @@ internal PackageSourceMoniker SelectedSource
             }
 
             _missingPackageStatus = false;
+
+            var gen = PackageList._list.ItemContainerGenerator;
+            gen.StatusChanged += Gen_StatusChanged;
+        }
+
+        private void PackageList_LoadItemsCanceled(object sender, EventArgs e)
+        {
+            if (!_loadCts.IsCancellationRequested)
+            {
+                NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+                {
+                    await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync(_loadCts.Token);
+                    _loadCts.Token.ThrowIfCancellationRequested();
+                    //Invalidate cache.
+                    Model.CachedUpdates = null;
+                    ResetTabLoadFlags();
+                    _loadCts.Cancel();
+                    //Load was cancelled, so trigger a reload of the current tab.
+                    PackageManagerLoaded(sender, new RoutedEventArgs());
```

Comment:
Basically just want to do a regular old reload because the user interrupted the fast tab-switch. 
Essentially hit the "Refresh" button... but I liked loaded method better.